indico/htdocs/js/indico/modules/categories/categories.js

```diff
@@ -111,4 +111,90 @@
+
+        function createCategoryNavigator(element, data) {
+            $('<div>').categorynavigator({
+                openInDialog: true,
+                onAction: function(category) {
+                    $.ajax({
+                        url: element.data('href'),
+                        type: 'POST',
+                        data: JSON.stringify($.extend({'category_id': category.id}, data || {})),
+                        dataType: 'json',
+                        contentType: 'application/json',
+                        error: handleAjaxError,
+                        success: function(data) {
+                            if (data.success) {
+                                location.reload();
+                            }
+                        }
+                    });
+                }
+            });
+        }
+
+        $('.js-move-event-to-subcategory').on('click', function(evt) {
+            createCategoryNavigator($(this))
+        });
+
+        $('.js-move-events-to-subcategory').on('click', function(evt) {
+            var $this = $(this);
+            var data = {};
+            if ($this.data('params') && $this.data('params').all_selected) {
+                data['all_selected'] = true;
+            } else {
+                data['event_id'] = _.map($('#event-management input[name=event_id]:checkbox:checked'), function(obj) {
+                    return obj.value;
+                });
+            }
+
+            createCategoryNavigator($this, data);
+        });
+
+        function deselectRows() {
+            $('#selection-message').hide();
+            $('.js-enabled-if-checked').data('params', {all_selected: false});
+        }
+
+        $('#event-management input[name=event_id]').on('change', function() {
+            if (!this.checked) {
+                deselectRows();
+            }
+        });
+
+        $('[data-select-all]').on('click', function() {
+            var $this = $(this);
+            var total = $this.data('total');
+            var isPaginated = $this.data('is-paginated');
+            var selectionMessage = $('#selection-message');
+            var selected = $($this.data('select-all'));
+
+            if (isPaginated && selected.length < total) {
+                var html = $('<span>', {
+                    text: $T.gettext('You have selected {0} out of {1} events. ').format(selected.length, total)
+                });
+                var selectAll = $('<a>', {
+                    'href': '#',
+                    'text': $T.gettext('Click here to select them all.'),
+                    'on': {
+                        click: function(evt) {
+                            evt.preventDefault();
+                            $('.js-enabled-if-checked').data('params', {all_selected: true});
+                            selectionMessage.text($T.gettext('You have selected all {0} events').format(total));
```

Comment:
@pferreir  did you forget "events" there? I don't think adding "event" dynamically to a translated thing would be a good idea